src/plugins/blocksmith/components/google-map-block-modal/getGoogleMapCoordinates.js

```diff
@@ -0,0 +1,26 @@
+/* globals GOOGLE_MAPS_API_KEY */
+const getGoogleMapCoordinates = async (str) => {
+  const url = `https://maps.googleapis.com/maps/api/geocode/json?address=${encodeURIComponent(str)}&key=${GOOGLE_MAPS_API_KEY}`;
+  let data = {};
+
+  try {
+    const response = await fetch(url);
+    data = await response.json();
+  } catch (error) {
+    data.status = error;
+  }
+
+  return new Promise((resolve) => {
+    const status = data.status;
+    if (status !== 'OK' || data.results.length === 0) {
+      console.error('Geocoding failed:', status);
+      resolve(null);
+      return;
+    }
+
+    const results = data.results[0];
+    resolve(`${results.geometry.location.lat},${results.geometry.location.lng}`);
```

Comment:
@bhsiao55 - move this line to _try_ block, so the exception is caught if the Google Maps API response doesn't have any results.

Reply:
yup makes sense